admin-dev/themes/new-theme/js/pages/product/components/dropzone/DropzoneWindow.vue

```diff
@@ -245,8 +246,10 @@
     mounted() {
       window.prestaShopUiKit.initToolTips();
       // We set the intial value of the first item in order to use the computed
-      this.captionValue = this.selectedFile.legends;
-      this.coverData = this.selectedFile.is_cover;
+      if (this.selectedFile) {
+        this.captionValue = this.selectedFile.legends;
```

Comment:
Is this one of the case where we could use?
```suggestion
        this.captionValue = this.selectedFile?.legends;
```

Reply:
No because we don't want to assign undefined to captionValue or coverData